data/changelog/opam/2018-05-22-opam-2-0-0-rc2.md

```diff
@@ -17,7 +17,7 @@ Among the new features, we have squeezed in full sandboxing of package commands
 
 The new release candidate also offers the possibility to setup a hook in your shell, so that you won't need to run `eval $(opam env)` anymore. This is specially useful in combination with local switches, because with it enabled, you are guaranteed that running `make` from a project directory containing a local switch will use it.
 
-The documentation has also been updated, and a preview of the opam 2 webpages can be browsed at https://opam.ocaml.org/2.0-preview/ (please report issues [here](https://github.com/ocaml/opam2web/issues)). This provides the list of packages available for opam 2 (the `2.0` branch of [opam-repository](https://github.com/ocaml/opam-repository/tree/2.0.0)), including the [compiler packages](https://opam.ocaml.org/2.0-preview/packages/ocaml-base-compiler/).
+The documentation has also been updated, and a preview of the opam 2 webpages can be browsed at https://opam.ocaml.org/2.0-preview/ (please report issues [here](https://github.com/ocaml/opam2web/issues)). This provides the list of packages available for opam 2 (the `2.0` branch of [opam-repository](https://github.com/ocaml/opam-repository/tree/2.0.0 - [404 Not Found])), including the [compiler packages](https://opam.ocaml.org/2.0-preview/packages/ocaml-base-compiler/ - [404 Not Found]).
```

Comment:
```suggestion
The documentation has also been updated, and a preview of the opam 2 webpages can be browsed at https://opam.ocaml.org/2.0-preview/ (please report issues [here](https://github.com/ocaml/opam2web/issues)). This provides the list of packages available for opam 2 (the `2.0` branch of [opam-repository](https://github.com/ocaml/opam-repository)), including the [compiler packages](https://opam.ocaml.org/packages/ocaml).
```

Updated opam repo link, even though it's now for 3.0

second: I _think_ the OCaml base compiler (??)